### DISCUSSION TITLE
Allow for easier file names Any extension allowed (hda/dsk/xxx)

### DISCUSSION THREAD

**Discussion by aihk** [erichelgeson/BlueSCSI-v1#108] [Q&A]
Not all formats are supported. The DSK I made with Mini vMac is not recognized properly

**Comment by erichelgeson** [comment]
Mini vMac only supports HFS Partitions - a real hard drive has a Partition Map, SCSI Driver, and HFS partition. Mini vMac has said they may support real drive images in the future. 

Note any file name extensions currently works as there are no standards around them, we just normally align with rasci's hda.

You can find more info about supported emulators here https://github.com/erichelgeson/BlueSCSI/wiki/Transferring-files-(Mac)

**Comment by aihk** [comment]
Thanks a lot for the warm reply! I will keep following you.

**Comment by aihk** [comment]
Sadly I can't use Basilisk II to make Macintosh Plus supported images

**Reply by erichelgeson** [reply]
Why is that? The premade system 6 images that support mac plus were made in Basilisk II. Note you can not boot into system 6 in basilisk ii, but you can boot off a 7 image and install system 6 onto a blank.

**Reply by aihk** [reply]
When Macintosh Plus uses BlueSCSI, there are 2 images that make a sizzling noise and can't get into the system

**Reply by erichelgeson** [reply]
Then the image you based it off of has the Lido driver which is unsupported on the Mac Plus - see the docs on the Mac Plus - https://github.com/erichelgeson/BlueSCSI/wiki/Supported-Systems#plus

Use an image or blank that does not use the Lido drivers.

**Reply by aihk** [reply]
Yes, I use the pre-made images to get into the system fine. The images I made myself would make sizzling noises. I think it might be the ROM. I also looked at the way they made it, with 2 HD's installed in the plus, and when I try that, it makes noise.

**Reply by aihk** [reply]
In BlueSCSI I put a prefab image, a blank image. Makes noise when starting up

**Reply by erichelgeson** [reply]
We have tested with all 3 known Plus rom's and they work fine. If you are experiencing the issue with 2 drive images it may be you have different drivers on each image. A work around would be to take the working image, duplicate it, and format it on real hardware. SCSI drivers are quite an annoyance when sharing images between macs as they do not all work across all machines.

**Reply by erichelgeson** [reply]
> In BlueSCSI I put a prefab image, a blank image. Makes noise when starting up

If you could zip up and upload the image causing issues I can inspect it and let you know what driver it is using.

**Reply by aihk** [reply]
https://airportal.cn/958878/8IoZYUCYBp    password   958878

**Reply by erichelgeson** [reply]
I get a connection timeout going to that site unfortunately.

**Reply by aihk** [reply]
I want to make an image of Chinese 7.5.3. Is it because I can't Chinese, but I can't make English.

**Reply by aihk** [reply]
https://www.mediafire.com/file/me6ce76ns73fbig/HD20-50.zip/file

**Reply by aihk** [reply]
Can you help me make a Chinese image to run on PLUS. https://winworldpc.com/product/mac-os-7/75

**Reply by erichelgeson** [reply]
The `HD20-50.hda` file has the Lido driver - use a different image to start with rather than one based on Lido.

**Reply by aihk** [reply]
https://winworldpc.com/download/3ec3abc3-bfc2-ac18-c39a-11c3a4e284a2

**Reply by aihk** [reply]
Or can you provide an image without the Lido driver

**Reply by erichelgeson** [reply]
On the Usage wiki page - there is a link to a mega drive with blanks/premade images/etc - should be everything you need.

https://github.com/erichelgeson/BlueSCSI/wiki/Usage#premade-images

**Reply by aihk** [reply]
thanks, i'll learn

**Reply by aihk** [reply]
Thanks a lot, it worked. It's an image problem.
